include/swift/Driver/DependencyGraph.h

```diff
@@ -153,6 +163,12 @@ class DependencyGraphImpl {
     (void)newlyInserted;
   }
 
+  /// Starting at \p node, visit the transitive closure of every dependent node.
+  /// Assume that the starting node is "cascading"; that all dependencies must
+  /// be dirty if the start is dirty. Therefore, mark the start. For each
+  /// visited node, add it to visited, and mark it if it cascades. The start
+  /// node is NOT added to visited.
```

Comment:
There's no comment here because there *is* a comment on the templated version. Can you keep them all together?

Reply:
Yes! Moved this comment and added an attempt at a cross-reference.